src/Microsoft.TestPlatform.Common/Logging/TestLoggerManager.cs

```diff
@@ -160,8 +161,11 @@ public void UpdateLoggerList(string argument, string loggerIdentifier, Dictionar
         /// <summary>
         /// Initializes all the loggers passed by user
         /// </summary>
-        public void InitializeLoggers()
+        /// <param name="requestData">Request Data for Providing Common Services/Data for Discovery and Execution</param>
+        public void InitializeLoggers(IRequestData requestData)
         {
+            var loggersUsed = new List<string>();
```

Comment:
>loggersUsed [](start = 16, length = 11)

We can avoid this, can we use InitializedLoggers instead ?

Reply:
Done